lib/vanagon/utilities.rb

```diff
@@ -281,5 +281,16 @@ def erb_file(erbfile, outfile = nil, remove_orig = false, opts = { :binding => b
       FileUtils.rm_rf erbfile if remove_orig
       outfile
     end
+
+    # Final
+    # - 1.0.0
+    # - 3.0.5.1
+    # - 4.3.0-5
+    # Nonfinal
+    # - 1.0.0-34-g1sadjf9283
+    # - 4.2.1.2344.gfdqw83rh
+    def final_version?(version)
+      !version.match(/g[a-f0-9]{7}$/)
```

Comment:
hmm. I guess that makes sense, but it still feels weird to me to just be matching on one specific thing we don't want to call final. For example, the fact that '1.0.0-g1234567' is not a final version, but '1.0.0-g123456789' IS is strange. But I'm definitely open to ideas for how to do this better.